dbus/com.redhat.Yggdrasil1.xml

```diff
@@ -8,6 +8,16 @@
         programmatic interaction with the yggdrasil system service.
     -->
     <interface name="com.redhat.Yggdrasil1">
+        <!--
+            GetClientID:
+            @id: The current client id on the interface.
+
+            Returns the current client id on the interface.
+        -->
+        <method name="GetClientID">
```

Comment:
Did you try defining this as a D-Bus object property rather than a "getter" method?